docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -643,18 +723,18 @@ Description of Workflow Tasks
 .. note::
    This section gives a general overview of workflow tasks. To begin running the workflow, skip to :numref:`Step %s <RocotoRun>`
 
-:numref:`Figure %s <WorkflowTasksFig>` illustrates the overall workflow. Individual tasks that make up the workflow are specified in the ``FV3LAM_wflow.xml`` file. :numref:`Table %s <WorkflowTasksTable>` describes the function of each task. The first three pre-processing tasks; ``MAKE_GRID``, ``MAKE_OROG``, and ``MAKE_SFC_CLIMO`` are optional. If the user stages pre-generated grid, orography, and surface climatology fix files, these three tasks can be skipped by adding the following lines to the ``config.sh`` file before running the ``generate_FV3LAM_wflow.sh`` script: 
+:numref:`Figure %s <WorkflowTasksFig>` illustrates the overall workflow. Individual tasks that make up the workflow are specified in the ``FV3LAM_wflow.xml`` file. :numref:`Table %s <WorkflowTasksTable>` describes the function of each baseline task. The first three pre-processing tasks; ``MAKE_GRID``, ``MAKE_OROG``, and ``MAKE_SFC_CLIMO`` are optional. If the user stages pre-generated grid, orography, and surface climatology fix files, these three tasks can be skipped by adding the following lines to the ``config.sh`` file before running the ``generate_FV3LAM_wflow.sh`` script: 
 
 .. code-block:: console
 
-   RUN_TASK_MAKE_GRID=”FALSE”
-   RUN_TASK_MAKE_OROG=”FALSE”
-   RUN_TASK_MAKE_SFC_CLIMO=”FALSE”
+   RUN_TASK_MAKE_GRID="FALSE"
+   RUN_TASK_MAKE_OROG="FALSE"
+   RUN_TASK_MAKE_SFC_CLIMO="FALSE"
 
 
 .. _WorkflowTasksFig:
 
-.. figure:: _static/FV3LAM_wflow_flowchart.png
+.. figure:: _static/FV3LAM_wflow_flowchart_v2.svg
```

Comment:
@chan-hoo Do you have a .png image of the updated FV3LAM_wflow_flowchart.png diagram? I have a PPT with the most updated ones in a slide, but it's sized differently from all the other ones and has your name at the bottom (I suppose we can keep your name in if you want, but the other .pngs in the docs don't have your name under them ;) )